### PR TITLE
Improve logging for test-cmd.sh

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
[test]

**Comment by stevekuznetsov** [comment]
:+1:  to this... finding the log in the end-to-end directory was awful

File: hack/test-cmd.sh [hunk 1]
```diff
@@ -153,7 +155,7 @@ os::util::sed "s/:7001$/:${ETCD_PEER_PORT}/g" ${SERVER_CONFIG_DIR}/master/master
 OPENSHIFT_ON_PANIC=crash openshift start master \
   --config=${MASTER_CONFIG_DIR}/master-config.yaml \
   --loglevel=4 \
```
Comment (stevekuznetsov) [inline hack/test-cmd.sh]: Shouldn't we bump this up if the added logging is all `V(5)`?

Comment (liggitt) [inline hack/test-cmd.sh]: weird, I had that at one point, must have gotten dropped

**Comment by liggitt** [comment]
[merge]

**Pull request merged**